src/HttpTrigger/tests/index.test.js

```diff
@@ -0,0 +1,12 @@
+var chai = require('chai');
+var httpFunction = require('../index')
+const context = require('../testing/defaultContext')
+var expect = chai.expect;
+	it ('Http trigger should return known text', async function() {
```

Comment:
Can you fix the indentation on this `if` block, please?

Also, can you use an arrow function here?

Reply:
Done.